src/Microsoft.DotNet.Interactive.ExtensionLab/Inspector/CSharpDecompiler/CSharpDecompiler.cs

```diff
@@ -15,7 +15,10 @@ internal static string Decompile(in PEFile assembly, in InspectionOptions inspec
         {
             var decompilationLanguageVersion = Defaults.GetDecompilationLanguageVersion(inspectionOptions.DecompilationLanguage);
 
-            var decompiler = new ICSharpCode.Decompiler.CSharp.CSharpDecompiler(module: assembly, assemblyResolver: new UniversalAssemblyResolver($"{Defaults.InternalAssemblyName}.dll", true, "netcoreapp3.1"), new DecompilerSettings(decompilationLanguageVersion))
```

Comment:
See my comments at https://github.com/icsharpcode/ILSpy/issues/2228#issuecomment-731844279... UniversalAssemblyResolver does not support the TFM format as used by the project system. It uses the format used by `System.Runtime.Versioning.TargetFrameworkAttribute`. You should use `.NETCoreApp,Version=3.1` instead of `".netcoreapp3.1"`.

Reply:
Thank you for following up.  I added a reply to the original issue and using `.NETCoreApp,Version=v5.0` was the proper solution.